modules/nodes-automatically-scaling-machines-to-available-bare-metal-hosts.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-automatically-scaling-machines-to-available-bare-metal-hosts_{context}"]
+= Automatically scaling machines to the number of available bare metal hosts
+
+You can configure `Machine` objects to be automatically created to match the number of available `BareMetalHost` objects. This is done by adding a `metal3.io/autoscale-to-hosts` annotation to the `MachineSet` object that you want to be scaled automatically.
+
+.Prerequisites
+
+* Install {op-system} bare metal compute machines for use in the cluster, and create corresponding `BareMetalHost` objects.
+* Install the {product-title} CLI (oc).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Edit the machine set that you want to configure for automatic scaling by running the following command, replacing `<machineset>` with the name of the machine set to be changed:
++
+[source,terminal]
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
+
+. Add the `metal3.io/autoscale-to-hosts` annotation, and set the number of machines that you want to scale to, for example:
```

Comment:
The value of the annotation is ignored and can be anything. From [the README](https://github.com/openshift/cluster-api-provider-baremetal/#machineset-scaling): "If you would like a MachineSet to be automatically scaled to the number of matching BareMetalHosts, annotate that MachineSet with key metal3.io/autoscale-to-hosts and any value."

The point is that instead of picking a number, it will scale the MachineSet to match however many BareMetalHosts there are.